dashboard/migrations/0016_submission_submitted_at.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.14 on 2020-07-29 00:34
```

Comment:
This migration needs to get bumped back to 18.

16 is used by #993 
17 is used by #981

Reply:
Now this migration number needs to be increased to at least 19, because [`0018_adding_mysql_cache.py`](/tl-its-umich-edu/my-learning-analytics/blob/master/dashboard/migrations/0018_adding_mysql_cache.py) was added by #982.

Reply:
Done. Renamed as 0020_submission_submitted_at.py now